godal.go

```diff
@@ -1710,6 +1710,81 @@ func (ra ResamplingAlg) rioAlg() (C.GDALRIOResampleAlg, error) {
 	}
 }
 
+// Gridding algorithms
+type GriddingAlg int
+
+const (
+	InverseDistanceToAPower GriddingAlg = iota + 1
+	MovingAverage
+	NearestNeighbor
+	MetricMinimum
+	MetricMaximum
+	MetricRange
+	MetricCount
+	MetricAverageDistance
+	MetricAverageDistancePts
+	Linear
+	InverseDistanceToAPowerNearestNeighbor
+)
```

Comment:
Given that GriddingAlg is never used by the public API, it should not be exposed publicly either. i.e. 's/GriddingAlg/griddingAlg`